applications/GeoMechanicsApplication/custom_processes/apply_k0_procedure_process.cpp

```diff
@@ -45,8 +42,8 @@ void SetConsiderDiagonalEntriesOnlyAndNoShear(ModelPart::ElementsContainerType&
 namespace Kratos
 {
 
-ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(ModelPart& model_part, const Parameters& rK0Settings)
-    : Process(Flags()), mrModelPart(model_part), mSettings(rK0Settings)
+ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(ModelPart& model_part, Parameters rK0Settings)
```

Comment:
Since the pass-by-reference has been replaced by pass-by-value, we should change the parameter name accordingly (Kratos Style Guide):
```suggestion
ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(ModelPart& model_part, Parameters K0Settings)
```

Reply:
Done